include/glaze/ext/jsonrpc.hpp

```diff
@@ -49,7 +49,7 @@ namespace glz::rpc
 namespace glz::rpc
 {
    using id_t = std::variant<glz::json_t::null_t, std::string_view, std::int64_t>;
-   static constexpr std::string_view supported_version{"2.0"};
+   inline constexpr std::string_view supported_version{"2.0"};
```

Comment:
`static constexpr` implies `inline`, so it's better here to stick with `static constexpr`.